src/main/java/org/fundacion/jala/converter/security/util/JwtUtil.java

```diff
@@ -64,8 +64,8 @@ public Date extractExpiration(final String token) {
     /**
      * Gets all the claims from a token.
      *
-     * @param token a string with the token
-     * @return the claims
+     * @param token a string with the token.
+     * @return Claims object with the claims.
```

Comment:
It should be "@return a Claims object...."

Reply:
ok dani, i will correct it